lxd/container_lxc.go

```diff
@@ -1567,6 +1567,13 @@ func (c *containerLXC) initLXC(config bool) error {
 				if err != nil {
 					return err
 				}
+			} else if m["nictype"] == "ipvlan" {
```

Comment:
Rather than inject after start which is racy, it's usually preferred to pre-create those devices on the host and pass them as type physical to the container. I believe that's what we already do for SRIOV.

Reply:
fixed in https://github.com/s3rj1k/lxd/commit/1a94452ee6fb0ddff524a2b55f619f1eb9c1ca31